R/Utilities.R

```diff
@@ -267,3 +260,36 @@ update_pkgs_logfun = function(repo, pkg = "NA" ) {
         fun(pkg, msg)
     }
 }
+
+## Remove a package from a GRANRepository object.
```

Comment:
@vobencha 

Thanks.

Please add comments to the effect that doing this in a non-production setting is not advised as it goes beyond suspending the package to destroying switchr's ability to find old versions of packages, and that suspending a package is preferable whenever possible and should be sufficient in most cases where the SCM in the manifest still exists.

Other than that, this looks ok